riocli/deployment/model.py

```diff
@@ -115,6 +115,32 @@ def create_object(self, client: Client, **kwargs) -> typing.Any:
                 provision_config.add_rosbag_job(component_name,
                                                 self._form_rosbag_job(req_job))
 
+        if 'features' in self.spec:
```

Comment:
@amitsingh21 We will also need the features section in the deployment JSONSchema.